src/components/NavBar/components/SubDrop/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/no-array-index-key */
+import styles from './styles.module.scss';
+
+interface Props{
+submenus:string[];
```

Comment:
Submenus here isn't a string array. That's why you have to use an any in the line 11. Assign a proper type and remove the any in line 11

Reply:
done